go.mod

```
@@ -1,11 +1,11 @@
 module github.com/BESTSELLER/dependabot-circleci
 
-go 1.21
+go 1.21.8
```

Would 1.22 not work for this one?
```suggestion
go 1.22
```

---

if you decide to bump to 1.22, remember to run `go mod tidy`